alien4cloud/alien4cloud_structs.go

```diff
@@ -189,11 +189,11 @@ type SimpleMark struct {
 	Column int `json:"column,omitempty"`
 }
 
-// searchRequest is the representation of a request to search objects as tpologies, orchestrators in the A4C catalog
-type searchRequest struct {
+// SearchRequest is the representation of a request to search objects such as topologies, orchestrators in the A4C catalog
+type SearchRequest struct {
 	Query string `json:"query,omitempty"`
-	From  string `json:"from"`
-	Size  string `json:"size"`
+	From  int    `json:"from,omitempty"`
```

Comment:
This seems to be a breaking change for clients.
It should at least be mentioned somewhere

Reply:
Yes, will add a breaking changes section in the milestone release notes

Reply:
searchRequest was actually private before this commit

Reply:
Ha right! Sorry.